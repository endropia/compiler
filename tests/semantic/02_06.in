var
	i: char;
begin
	i := 'a';
	while (i > 2) do begin
	i += 1;
	end;
end.