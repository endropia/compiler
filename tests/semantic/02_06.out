(4, 4) ASSIGN assigment operation is not overloaded for char and string